### DISCUSSION TITLE
iTerm port

### DISCUSSION THREAD

**Discussion by fedcho** [primer/github-vscode-theme#304] [Ideas]
For your consideration:
https://github.com/fcaldera/github-primer-iterm2

**Comment by cdalvaro** [comment]
I've made this port: [cdalvaro/github-theme-iterm](https://github.com/cdalvaro/github-theme-iterm) by exporting terminal colors from VSCode with command: `Developer: Generate Color Theme From Current Settings`.

The result is slightly different to [fcaldera/github-primer-iterm2](https://github.com/fcaldera/github-primer-iterm2), but I think is more accurate to _github-vscode-theme_ when you compare the VSCode terminal colors with _cdalvaro/github-theme-iterm_

![GitHub Theme for iTerm](https://github.com/cdalvaro/github-theme-iterm/raw/main/images/GitHubThemeiTerm.png)

You can download the theme for macOS Terminal app too.
